lib/operations/common_functions.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+function defineAspects(operation, aspects) {
```

Comment:
nit: aspects are defined in `OperationBase`, should `defineAspects` live there as well?